main/EventSystem.cpp

```diff
@@ -1235,7 +1254,28 @@ void CEventSystem::EvaluateEvent(const std::string &reason, const uint64_t Devic
 		{
 			if (reason == "device" && filename.find("_device_") != std::string::npos)
 			{
-				EvaluateLua(reason, lua_Dir + filename, "", DeviceID, devname, nValue, sValue, nValueWording, 0);
+				bDeviceFileFound = false;
```

Comment:
This changes the way Domoticz loads scripts and can break existing setups.
For instance, if you have one script dealing with multiple devices, it will stop working as before as soon as the script has the name of a device.
For instance I'm having a script named "script_device_garage" to deal with everything with the garage (sensor to know if it's open, zwave module to open the door, and a generic fake device named 'garage'). This stopped working as the script won't be called for anything but the fake device.

If this is intended - we should modify https://www.domoticz.com/wiki/Events#By_adding_your_own_scripts_to_the_scripts.2Flua.2F_dir_in_your_domoticz_installation_folder to document this new behavior.
Thx !